features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX/CyH4TransportDriver.h

```diff
@@ -93,6 +94,7 @@ class CyH4TransportDriver : public cordio::CordioHCITransportDriver {
 
     DigitalInOut bt_host_wake;
     DigitalInOut bt_device_wake;
+    bool bt_host_wake_active;
```

Comment:
This value should be initialised in the constructor.

Reply:
Thanks for review feedback. Addressed the feedback with new commit. Please review the changes and approve the PR Merge request.